### ISSUE TITLE
There is no way to semantically markup "incidental text"

### ISSUE THREAD

**Issue by dylanb** [w3c/html-aria#106]
### Issue
[WCAG 1.4.3 exempts text from color contrast requirements inter alia if the text is "incidental"](https://www.w3.org/TR/WCAG20/#visual-audio-contrast), there is no way in HTML or ARIA (or any combination of the two) to mark up text as incidental.

I have seen the following markup being incorrectly used in the wild:

```
<tag role="presentation" aria-hidden="true">Incidental Text</tag>
```

This does not work because a role of `none` or `presentation` is also used to remove native semantics (such as marking a table as a layout table) and `aria-hidden="true"` still leaves the text visible and therefore in scope for sighted users.

### Potential future solution: using `inert` and `aria-hidden`

```
<tag inert aria-hidden="true">Incidental Text</tag>
```

The advantage is it relies only on features that are either already in existence or proposed. We would then have to document this usage. The disadvantage is the semantics are being inferred from the combination of two features whose use together might also be for other reasons.

### Potential solution: new role

```
<tag role="incidental">Incidental Text</tag>
```

Advantage of this approach is that the semantics are clear. The disadvantage is that it requires a new feature.

**Comment by mcking65** [comment]
confused ... are screen readers supposed to ignore incidental text? Seems like a dangerous kind of concept ... where is incidental text defined?

If there were to be a new feature for something like this, it would likely be better as a property, not a role.

**Comment by dylanb** [comment]
yes, incidental text is supposed to be ignored by screen readers because it is "pure decoration" https://www.w3.org/TR/WCAG20/#puredecdef

**Comment by scottaohara** [comment]
Going to close this as it's been quite some time, and doesn't seem this specification is the place for coming up with ideas for new roles.
